OpenRA.Game/Settings.cs

```diff
@@ -159,6 +159,8 @@ public class GameSettings
 		public string Mod = "modchooser";
 		public string PreviousMod = "ra";
 
+		public Dictionary<string, string> LastFaction = new Dictionary<string, string>();
```

Comment:
This will eventually want to move into mod-specific settings, but I don't have a big problem with putting this here for now.